presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/RemoveRedundantTopN.java

```diff
@@ -25,9 +25,8 @@
 
 /**
  * Replace TopN node
- * 1. With a Sort node when the subplan is guaranteed to produce fewer rows than N
- * 2. With its source when the subplan produces only one row
- * 3. With a empty ValuesNode when N is 0
+ * 1. With its source when the subplan is at most one row
```

Comment:
this change should be part of the commit that adds this class

Reply:
ok will move to the 2nd commit, along with its corresponding test